specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/examples/StorageAccountCreateObjectReplicationRestorePointMarker.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "res7687",
+    "accountName": "src1122",
+    "objectReplicationPolicyId": "2a20bb73-5717-4635-985a-5d4cf777438f",
```

Comment:
Where can customer get the ID? Should customer generate a random UUID from client side, and send to server?
We don't see any other SRP API ask customer to generate a UUID and input to server.
There are many APIs will generate UUID from server side and sent back to customer.
So it might be better don't let customer to input the UUID, but generate one from server and send back to customer.